content/en/docs/Reference/configuration.md

```diff
@@ -157,8 +166,17 @@ reset:
   # if set to true it will format persistent partitions ('oem 'and 'persistent')
   reset-persistent: true
   reset-oem: false
-  
-  
+
+  # Creates these dirs in the rootfs during installation. As the rootfs is RO from boot, sometimes we find that we
```

Comment:
should this be "during reset" or is it because an installation is happening during reset?

Reply:
its just copy paste lol